developer_manual/app/schema.rst

```diff
@@ -1,12 +1,153 @@
-===============
-Database Schema
-===============
+===================
+Database Migrations
+===================
 
-.. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
+.. sectionauthor:: Thomas Müller <thomas.mueller@tmit.eu>
+
+ownCloud uses migration steps to perform any changes between versions. 
+In most cases, these changes are changes to the database schema, but it is not only limited to them.
+Therefore we support three kinds of migration steps: simple, SQL and schema.
+
+There are currently three supported migration types:
+
+- **Simple.** This allows general migration steps to be executed. These are quite similar to the `migration repair steps <https://doc.owncloud.org/api/classes/OCP.Migration.IRepairStep.html>`_.
+- **SQL.** This allows you to create a list of executable SQL commands.
+- **Schema.** This allows migration via schema migration operations.
+
+Starting with ownCloud 10, this is the preferred way to perform any kind of migrations and is enabled by default within core.
```

Comment:
the answer is here: https://github.com/owncloud/documentation/pull/2862#discussion_r100994570 😉